### PR TITLE
SLE 11 KDE yast2 users fix

### PR DESCRIPTION
**Pull request by dzedro**
Failing https://openqa.suse.de/tests/49520/modules/yast2_users/steps/3
I tried to find some amazing solution for this and this is my solution.
tested:
Opensuse KDE
http://dzedro.qa.suse.cz/tests/859/modules/yast2_users/steps/3
SLE11 KDE
http://dzedro.qa.suse.cz/tests/857/modules/yast2_users/steps/14
SLE12 GNOME
http://dzedro.qa.suse.cz/tests/860/modules/yast2_users/steps/3

### CODE REVIEW

**Comment by sysrich** [comment]
Can't see the links of it working, but.. looks to me that the reason you need to do this is a generic 'KDE' problem, and not specific to the yast2_users.pm
(my guess is we only see this problem because it's the first X11 test using xdg-su or other such 'not in the application finder' binary)

So the way I'd recommend would be to take this code out of here, and put it in a new test, eg. X11/sle11_kde_setup.pm which I'd then call in the SLE11 main.pm only, inside a if (kdestep_is_applicable) {}

This approach would also give you an opportunity to make the test simpler, you wouldn't need the "if check_screen 'kde-allow-any-command' ..", you can use an assert, as you'll *know* it should be loading the SLE11 KDE version of that launcher

TL;DR version - I think the code and the solution looks good, I just think yast2_users.pm is the wrong place for it.

**Comment by dzedro** [comment]
finally :)
tested: http://10.100.98.90/tests/868

**Comment by dzedro** [comment]
http://10.100.98.90/tests/883 #466
